core/src/com/unciv/ui/tutorials/TutorialController.kt

```diff
@@ -23,19 +24,23 @@ class TutorialController(
 
     private fun showTutorialIfNeeded() {
         val tutorial = tutorialQueue.firstOrNull()
-        if (tutorial != null && !isTutorialShowing) {
-            isTutorialShowing = true
-            val texts = tutorialMiner.getTutorial(tutorial, UncivGame.Current.settings.language)
-            tutorialRender.showTutorial(TutorialForRender(tutorial, texts)) {
-                tutorialQueue.remove(tutorial)
-                isTutorialShowing = false
-                with(UncivGame.Current.settings) {
-                    if (!tutorialsShown.contains(tutorial.name)) {
-                        tutorialsShown.add(tutorial.name)
-                        save()
+        if (tutorial == null) {
+            allTutorialsShowedCallback?.invoke()
+        } else {
```

Comment:
I think this should just be `else if`, the extra `{` and indentation just make it seem more complicated than it is

Reply:
fixed it